.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,20 @@
+
+📅 Target dates:
+
+Release Manager: 
+
+Release checklist:
```

Comment:
We should add merging the `goreleaser` PR in the [homebrew-tap](https://github.com/defenseunicorns/homebrew-tap) repo as a step in the release checklist